requirements/test.in

```diff
@@ -5,3 +5,4 @@
 
 pytest-cov                # pytest extension for code coverage statistics
 pytest-django             # pytest extension for better Django support
+xblock-sdk
```

Comment:
add blank line

Reply:
End of File Blank line missing in all files.